core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -99,7 +99,7 @@
     /**
      * Validate {@code sslContext} is configured properly. If {@code sslContext} is configured as client
      * context, or key store password is not given to key store when {@code sslContext} is created using key
-     * manager factory, the validation will fail and an {@link SSLException} will be raised.
+     * manager factory, the validation will fail and a {@link SSLException} will be raised.
```

Comment:
an is correct here. 'an es-es-el...'

Reply:
Oh.... I see :sob: :sob: :sob: 
Fixed it. Thank you for your patience.

Reply:
No worries :-D